packages/console/src/hooks/use-connector-in-use.ts

```diff
@@ -23,12 +23,14 @@ const useConnectorInUse = () => {
       const relatedIdentifier =
         type === ConnectorType.Email ? SignInIdentifier.Email : SignInIdentifier.Sms;
 
-      return (
-        data.signIn.methods.some(
-          ({ identifier, verificationCode }) => verificationCode && identifier === relatedIdentifier
-        ) ||
-        (data.signUp.identifiers.includes(relatedIdentifier) && data.signUp.verify)
+      const usedInSignUp =
+        data.signUp.identifiers.includes(relatedIdentifier) && data.signUp.verify;
+
+      const usedInSignIn = data.signIn.methods.some(
+        ({ identifier, verificationCode }) => verificationCode && identifier === relatedIdentifier
```

Comment:
On second thought, I don't think this logic is correct. This can only tell us that an email and/or sms connector is required in either sign-up or sign-in flow. But it can't tell us which email or which sms connector is currently in use.

Reply:
Since we only have one email/sms connector, they're in use when they're required in the sign up / sign in flow.

Reply:
OK, I understand it's also what the old code does, so let's move on.